migrations/2_deploy_organization.js

```diff
@@ -30,50 +31,48 @@ module.exports = async (deployer) => {
   const orgName = "Genesis";
```

Comment:
Will change the name to "Genesis-Test" or something which defer that from "Genesis"

Reply:
? this is the real genesis DAO, especially when we deploy to main net

Reply:
@orenyodfat can you please address Tibet's question, thanks!

Reply:
Going with Genesis for the demo tomorrow